### PR TITLE
Rename organization from base16-project to tinted-theming

### PR DESCRIPTION
**Pull request by JamyGolden**
Base16-project had announced that it was [going to change their name in July](https://github.com/tinted-theming/home/issues/51). [After a bit of effort](https://github.com/tinted-theming/home/issues/38), we've finally renamed the organization.

- Update references to base16-project
- Update the license
- Add scheme meta info to templates as a comment (it seems a `#` is how you add a comment in xfce4 `*.theme` files after searching online, let me know if I'm wrong)

### CODE REVIEW

File: templates/default.16.mustache [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Scheme name: {{scheme-name}}
+# Scheme author: {{scheme-author}}
+# Template author: Tinted Theming (https://github.com/tinted-theming)
```
Comment (afq984) [inline templates/default.16.mustache]: nit: these probably belong to a separate commit

File: templates/default.mustache [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# Scheme name: {{scheme-name}}
+# Scheme author: {{scheme-author}}
+# Template author: Tinted Theming (https://github.com/tinted-theming)
```
Comment (afq984) [inline templates/default.mustache]: nit: these probably belong to a separate commit

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2018 Yu, Li-Yu
+Copyright (c) 2022 [Tinted Theming](https://github.com/tinted-theming)
```
Comment (afq984) [inline LICENSE]:
```suggestion
Copyright (c) 2022 Tinted Theming (https://github.com/tinted-theming)
```

This LICENSE is not rendered as Markdown on GitHub

**Review by afq984 — CHANGES_REQUESTED** [review]

File: templates/default.16.mustache (hunk 1 above)
Comment (JamyGolden) [reply templates/default.16.mustache]: I've moved it to this PR https://github.com/tinted-theming/base16-xfce4-terminal/pull/14

**Comment by JamyGolden** [comment]
Updated

File: LICENSE (hunk 3 above)
Comment (JamyGolden) [reply LICENSE]: :+1: Chris did this originally and I just copied that markdown style, but it seems to be plain text so doesn't make sense to use markdown formatting.

**Review by afq984 — APPROVED** [review]
